delegation/src/Pos/Delegation/Logic/VAR.hs

```diff
@@ -71,10 +71,12 @@ type ReverseTrans = HashMap StakeholderId (HashSet StakeholderId, HashSet Stakeh
 -- Takes a set of dlg edge actions to apply and returns compensations
 -- to dlgTrans and dlgTransRev parts of delegation db. Should be
 -- executed under shared Gstate DB lock.
+--
+-- eActions list passed must be effectively a set.
```

Comment:
Why did you change it from a set to a list then?

Reply:
Because there was an extra conversion.

Reply:
Okay, then I would prefer seeing it in a separate commit, but it's minor, doesn't really matter.